### DISCUSSION TITLE
Building libfluidsynth-dev

### DISCUSSION THREAD

**Discussion by Utkarsh-AS** [FluidSynth/fluidsynth#1190] [Q&A] [answered]
Hi, I am trying to build qsynth using the latest fluidsynth. The dev package of fluidsynth available in the repository is an older one so I tried building fluidsynth from source following the instructions given here. The build completed successfully but when I install it, it only installs fluidsynth.
My question is how can I get the libfluidsynth-dev? What am I missing?
Ps. I am not new to Linux in general but I am new to building from source, so I might be missing something very simple.

**Comment by derselbst** [comment, marked as answer]
> The build completed successfully but when I install it, it only installs fluidsynth.

`libfluidsynth-dev` is a subset of a complete fluidsynth installation. Since you have installed fluidsynth successfully, you don't need any `libfluidsynth-dev` or other `*fluidsynth*` package provided by your distro.

**Reply by Utkarsh-AS** [reply]
Thanks for replying, I was able to successfully build and install qsynth.
